### DISCUSSION TITLE
Why i got a ban at Revolt Lounge?

### DISCUSSION THREAD

**Discussion by lin-calvin** [stoatchat/discussions#365] [Q&A] [answered]
I do noting wrong
My username is `calvinlin`

**Comment by sussycatgirl** [comment, marked as answer]
This is not the place for this. If you want to appeal a ban, send an email to lounge@revolt.chat.

**Reply by insertish** [reply]
I'm just going to add that your name does not come up as banned, are you on the latest client?
If not, CTRL + F5 / R to update.
